plugins/uefi/meson.build

```diff
@@ -15,6 +15,8 @@ shared_module('fu_plugin_uefi',
   fu_hash,
   sources : [
     'fu-plugin-uefi.c',
+    'fu-tpm-eventlog-common.c',
+    'fu-tpm-eventlog-device.c',
```

Comment:
I don't think stuffing it all in one plugin solves the architectural problem. There's nothing really update-capsule specific about the tpm event log, and this feels like working around the design rather than fixing the real issue. I also don't think the TPM event log is a logical child of the system firmware, it's a child of the TPM device itself. I'll have a think about a better way we can do this.

Reply:
The log itself is coming from firmware though, doesn't that make it a child of the firmware?

If it does go back into it's own plugin I would like to still see the functionality I added to fwupdate of spitting out a random PCR log in a separate tool though.

Reply:
I'm just prototyping exactly that :)